components/dashboard/src/settings/Account.tsx

```diff
@@ -39,7 +39,7 @@ export default function Account() {
             </div>
             <div className="flex justify-end mt-6">
                 <button className="text-gray-900 border-white bg-white hover:border-gray-200" onClick={close}>Cancel</button>
-                <button className={"ml-2 border-red-900 bg-red-500 disabled:opacity-50" + (typedEmail !== primaryEmail ? '' : ' hover:bg-red-700')} onClick={deleteAccount} disabled={typedEmail !== primaryEmail}>Delete Account</button>
+                <button className={"ml-2 warn disabled:opacity-50" + (typedEmail !== primaryEmail ? ' hover:bg-gray-100' : '')} onClick={deleteAccount} disabled={typedEmail !== primaryEmail}>Delete Account</button>
```

Comment:
**issue**: This should become a _primary_ button using the danger variant, not using the outlined version, as this it the single primary action (CTA) in this screen.

Reply:
Ah, so we would have `primary danger` (white on red bg) and `secondary danger` (red on white with red border)?

Reply:
Exactly! Also, _tertiary_ danger could be needed in the future. See borderless Cancel button (tertiary default) on some modals. 🎯